release_notes/Service_Fabric_ReleaseNotes_90CU1.md

```diff
@@ -56,7 +56,7 @@ The package Microsoft.ServiceFabric.AspNetCore.WebListener will no longer be shi
 
 | Versions | IssueType | Description | Resolution | 
 |-|-|-|-|
-| **Windows - 9.0.1028.9590<br>Ubuntu 16 - 9.0.1035.1<br>Ubuntu 18 - 9.0.1035.1** | **Bug** | Reliability and Security fix |**Brief desc**: Security fixes related to Container logs and runtime dependencies.<br> For container Logs, Service Fabric runtime on Linux will no longer process Service Fabric-generated diagnostic logs emitted by enlightened containerized workloads. On Windows, the logging agent will no longer use the container-mounted log folder to stage metadata files.<br> Customers will need to update any runtime dependencies with newer versions, which include fixes for known/published vulnerabilities.<br>  |
+| **Windows - 9.0.1028.9590<br>Ubuntu 16 - 9.0.1035.1<br>Ubuntu 18 - 9.0.1035.1** | **Bug** | Reliability and Security fix |**Brief desc**: Security fixes related to Container logs and runtime dependencies.<br> For container Logs, Service Fabric runtime on Linux will no longer process Service Fabric-generated diagnostic logs emitted by enlightened containerized workloads. On Windows, the logging agent will no longer use the container-mounted log folder to stage metadata files.<br> Customers will need to update any runtime dependencies with newer versions, which include fixes for known/published vulnerabilities.<br>Read more about it at - <ul><li>[Service Fabric Privilege Escalation from Containerized Workloads on Linux](https://msrc-blog.microsoft.com/2022/06/28/azure-service-fabric-privilege-escalation-from-containerized-workloads-on-linux/)</li></ul> | <br> Customers will need to update any runtime dependencies with newer versions, which include fixes for known/published vulnerabilities.<br>  |

```

Comment:
nit
The public disclosure detailing the issue has been made available vs just read more about it

Reply:
Fixed